src/Data/Skills/sup_int.lua

```diff
@@ -1644,6 +1644,12 @@ skills["MinionFocusFire"] = {
 	skillTypes = { [SkillType.Spell] = true, [SkillType.Instant] = true, [SkillType.Minion] = true, [SkillType.Triggerable] = true, [SkillType.Duration] = true, [SkillType.InstantShiftAttackForLeftMouse] = true, [SkillType.Cooldown] = true, },
 	statDescriptionScope = "debuff_skill_stat_descriptions",
 	castTime = 0,
+	statMap = {
+		["resist_all_%"] = {
+			mod("EnemyModifier", "LIST", { mod = mod("ElementalResist", "BASE", nil) }),
+			mod("EnemyModifier", "LIST", { mod = mod("ChaosResist", "BASE", nil) }),
+		},
```

Comment:
This doesn't seem to do anything from my testing...

Reply:
weird... This 
```
	["nearby enemies have (%-%d+)%% to all resistances"] = function(num) return {
		mod("EnemyModifier", "LIST", { mod = mod("ElementalResist", "BASE", num) }),
		mod("EnemyModifier", "LIST", { mod = mod("ChaosResist", "BASE", num) }),
	} end,
	```
	works in supporting `Crown of the Tyrant` unique. Not sure why it doesn't work for the gem

Reply:
After messing with an "enemy on consecrated ground" mod, I determined that `EnemyModifier` doesn't work for mods from skills.  You can fake it with `GlobalEffect` and `Debuff`, but that might cause other side effects I'm not sure about.

Reply:
I reached the same conclusion